src/main/resources/scripts/generate-results.py

```diff
@@ -103,26 +103,19 @@ def runManageWithWait(command_line, silent=False):
 # Determine if this version of VectorCAST supports new-style reporting/Data API
 def checkUseNewReportsAndAPI():
     if os.environ.get("VCAST_REPORT_ENGINE", "") == "LEGACY":
-        # Using legacy reporting with new reports - fall back to parsing html report
+        # The execution plugin will ignore this value, but warn user.
         if verbose:
             print("VectorCAST/Execution ignoring LEGACY VCAST_REPORT_ENGINE.")
 
-    # Look for existence of file that only exists in distribution with the new reports
-    check_file = os.path.join(os.environ.get('VECTORCAST_DIR'),
-                             "python",
-                             "vector",
-                             "apps",
-                             "ReportBuilder",
-                             "reports",
-                             "full_report.pyc")
-    if os.path.isfile(check_file):
+    try:
+        from vector.apps.ReportBuilder.reports import full_report

```

Comment:
Is this script run from `vpython`?

Reply:
I assume so, based on the imports at the top of the file

Reply:
Line 59 already imports from this package; set a global rather doing the import again

Reply:
After further investigation, the only other time the code in this repository references or runs this file is through `vpython`

Reply:
This will only work as long as the paths don't change, which isn't guaranteed.  All we really care about is if the DataAPI is importable.  That will remove both the reliance on checking the .pyc file and maintaining the import of a package that may move in the future.